### PR TITLE
PLT-3473 Made shortcuts only work for CMD on OSX

### PR DESCRIPTION
**Pull request by DavidLu1997**
Please make sure you've read the [pull request](http://docs.mattermost.com/developer/contribution-guide.html#preparing-a-pull-request) section of our [code contribution guidelines](http://docs.mattermost.com/developer/contribution-guide.html).

#### Summary
Previously, the following shortcuts worked with both CTRL and CMD on Mac, now they only work with CMD. Windows behaviour is unchanged, as in only CTRL works.

- Open a quick channel switcher dialog: CMD+K
- Open account settings: CMD+SHIFT+A
- Open recent mentions: CMD+SHIFT+M
- Upload a file: CMD+U

#### Issue/Ticket Link
https://mattermost.atlassian.net/browse/PLT-3473

### CODE REVIEW

File: webapp/utils/utils.jsx [hunk 1]
```diff
@@ -32,6 +32,20 @@ export function cleanUpUrlable(input) {
     return cleaned;
 }
 
+export function isMac() {
+    if (navigator.userAgent.indexOf('Mac') !== -1) {
```
Comment (enahum) [inline webapp/utils/utils.jsx]: Maybe should be better to check navigator.platform?

```javascript
export function isMac() {
  return navigator.platform.toUpperCase().indexOf('MAC') >= 0;
}
```

File: webapp/utils/utils.jsx [hunk 2]
```diff
@@ -32,6 +32,20 @@ export function cleanUpUrlable(input) {
...
+    }
+    return false;
+}
+
+export function cmdOrCtrlPressed(e) {
+    if ((isMac() && e.metaKey) || (!isMac() && e.ctrlKey)) {
+        return true;
+    }
+    return false;
+}
```
Comment (enahum) [inline webapp/utils/utils.jsx]: Why not like this

```javascript
export function cmdOrCtrlPressed(e) {
    return ((isMac() && e.metaKey) || (!isMac() && e.ctrlKey);
}
```

**Comment by DavidLu1997** [comment]
@enahum changed :)

**Comment by hmhealey** [comment]
+1

**Comment by enahum** [comment]
+1

**Comment by lfbrock** [comment]
Tested the commands listed, and they only work with CMD and not CTRL as expected. Didn't see any issues.

**Comment by jasonblais** [comment]
Have drafted a note for the changelog; docs already follow the behaviour submitted with this PR

**Pull request merged**